package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Practice_Assignments",
+  "version": "1.0.0",
+  "description": "This is practice assignment using campaign use case",
+  "main": "server.js",
+  "scripts": {
+    "start": "babel-node --presets es2015 server.js"
+  },
+  "keywords": [
+    "campaigns",
+    "express"
+  ],
+  "author": "Pawan Agine",
```

Comment:
For babel environment include the devDependencies and for Testing as well include libraries in the same. ex - `mocha`, `chai`